libs/design-system/button/src/lib/button-styles/button-styles.component.scss

```diff
@@ -41,14 +41,38 @@ $y-multiplier: 0.4566;
   }
 }
 
+// Custom Class for basic primary and secondary
+:is(a, button)[mat-button] {
+  &.button-size-large {
+    --mat-text-button-horizontal-padding: 1rem;
+    --mdc-text-button-container-height: 2.5rem;
+    font: var(--sys-label-large);
```

Comment:
Please also set letter-spacing whenever you set font, i.e. `letter-spacing: var(--sys-label-large-tracking);`